### DISCUSSION TITLE
Schlage lock FE599 stays in unlocked state even though it auto-relocked

### DISCUSSION THREAD

**Discussion by ibddot** [zwave-js/zwave-js#7511] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
My FE599 lock is now staying in "unlocked" state after keypack unlock operation. The lock auto-relocks itself but doesn't report that. Here is some other reference to this same issue: https://community.smartthings.com/t/automatic-poll-schlage-fe599nx-after-unlocked-with-code/50724. If I refresh the Basic V0 values for lock in the ZWaveJS UI, the value changes from 1 to 99 and HA then lists the lock in the locked state.

What did you expect to happen instead?
I expect state to show locked when the lock is locked without having to manually refresh the value.

  1. Unlock FE599 lock with the code on the keypad. 
  2. Wait for the lock to auto-relock in ~10 seconds.
  3. Check the state of the lock in HA or ZWaveJS and it shows unlocked even though it is locked.

Anything else we should know?
It didn't have this behavior until I recently updated and I believe it is due to https://github.com/zwave-js/node-zwave-js/pull/7122. While that mapping is correct, the behavior is not because the lock does not report the auto-relocking event. I would propose reverting the mapping of the keypad_unlock unless there is another way to update the status to locked again after the lock auto-relocks.

### Software versions

zwave-js-ui: 9.27.8
zwave-js: 14.3.7

Home Assistant
Z-Wave JS UI Current version: 3.18.0
Core 2024.12.3
Supervisor 2024.12.0
Operating System 14.1
Frontend 20241127.8

### Z-Wave Controller (Stick/Dongle/...)

HUSBZB-1 USB

### Device information

Manufacturer: Allegion
Model name: CKPD FE599
Node ID: 40

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave-js-ui-store.zip](https://github.com/user-attachments/files/18221241/zwave-js-ui-store.zip)

**Comment by AlCalzone** [comment, marked as answer]
Sorry for the delay. I merged your PR https://github.com/zwave-js/zwave-js/pull/7870 - the behavior didn't seem quite correct.
